module/VuFind/src/VuFind/Service/Factory.php

```diff
@@ -42,6 +42,274 @@
  */
 class Factory
 {
+    /**
```

Comment:
Looks like there was a merge glitch here, and a bunch of outdated and deleted factory code has come back. Rather than creating a static factory method here for the CSRF validator, it would probably be better to give it its own factory class like most VuFind services now have. If you want help fixing this, let me know and I can take care of it later in the week.